public/app/features/dashboard-scene/scene/new-toolbar/actions/HiddenElementsSwitch.tsx

```diff
@@ -0,0 +1,21 @@
+import { selectors } from '@grafana/e2e-selectors';
+import { t } from 'app/core/internationalization';
+
+import { ToolbarSwitch } from '../ToolbarSwitch';
+import { ToolbarActionProps } from '../types';
+
+export const HiddenElementsSwitch = ({ dashboard }: ToolbarActionProps) => (
```

Comment:
talked to patrick and we can remove this toggle, lets try if we can only visually style hidden elements in edit mode instead. I think we can get pretty far with that. maybe we add this this in the future but not needed for initial versions I think